src/Application/Commands/HandleOutOfSyncFunds.php

```diff
@@ -117,7 +126,7 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 
             $output->writeln("Funding {$funding->getId()} is under-matched by $undermatchAmount. $details");
 
-            if ($mode === 'fix') {
+            if ($mode === 'fix' || ($mode === '2023-11-30-patch' && in_array($funding->getId(), $problemIds, true))) {
```

Comment:
the in_array check here shouldn't be necessary but is just a nice defense in depth in case `findBy` did something we didn't expect right?

Reply:
Yep